tests/dapr_tests.mk

```diff
@@ -253,7 +253,7 @@ create-test-namespace:
 delete-test-namespace:
 	kubectl delete namespace $(DAPR_TEST_NAMESPACE)
 
-setup-3rd-party: setup-helm-init setup-test-env-redis setup-test-env-kafka setup-test-env-mongodb setup-test-env-zipkin
+setup-3rd-party: setup-helm-init setup-test-env-redis setup-test-env-kafka setup-test-env-mongodb setup-test-env-zipkin setup-test-env-cert-manager
```

Comment:
I would really prefer if we didn't deploy yet another Helm chart in our test K8s clusters... Makes tests slower, requires more CPU, and can add to flakiness when a chart fails to deploy due to network issues (happens quite frequently).

Can we find a way that doesn't involve using cert-manager? How about we create the certs in a script and load them as secrets in K8s before we start the tests?

Reply:
@ItalyPaleAle No worries. We can generate them before the tests is executed as a setup step. Is it possible to get a Kubernetes client configured in the e2e test runner?

Reply:
Yes- we can pass the Secret resources as a `ComponentDescription` to the test runner.